config/jenkins_infra.ci.jenkins.io.yaml

```diff
@@ -264,15 +264,15 @@ controller:
                     galleryImageVersion: "1.86.0"
                     galleryName: "prod_packer_images"
                     galleryResourceGroup: "prod-packer-images"
-                    gallerySubscriptionId: "${JENKINSINFRA_AZURE_SECONDARY_SUBSCRIPTION_ID}"
+                    gallerySubscriptionId: "^${JENKINSINFRA_AZURE_SECONDARY_SUBSCRIPTION_ID}"
```

Comment:
What is the reason of this change @jayfranco999 ? it is unrelated to your PR and will break things 🤔